dags/rlc.py

```diff
@@ -0,0 +1,159 @@
+            log_url=context.get('task_instance').log_url,
+        )
+    failed_alert = SlackWebhookOperator(
+        task_id='slack_test',
+        http_conn_id='slack',
+        webhook_token=slack_webhook_token,
+        message=slack_msg,
+        username='airflow',
+        proxy='http://137.15.73.132:8080'
+        )
+    return failed_alert.execute(context=context)
+
+# ------------------------------------------------------------------------------
+AIRFLOW_DAGS = os.path.dirname(os.path.realpath(__file__))
+AIRFLOW_ROOT = os.path.dirname(AIRFLOW_DAGS)
+AIRFLOW_TASKS = os.path.join(AIRFLOW_ROOT, 'assets/rlc/airflow/tasks')
+
+DEFAULT_ARGS = {
+    'email': ['Cathy.Nangini@toronto.ca'],
+    'email_on_failure': True,
+    'email_on_retry': True,
+    'owner': 'airflow',
+    'start_date': datetime(2019, 9, 16), # YYYY, MM, DD
+    'task_concurrency': 1,
+    'on_failure_callback': task_fail_slack_alert
+}
+
+# ------------------------------------------------------------------------------
+
+def pull_rlc(conn):
+    '''
+    Connect to bigdata RDS, pull Red Light Camera json file from Open Data API,
+    and overwrite existing rlc table in the vz_safety_programs_staging schema.
+    '''
+
+    local_table='vz_safety_programs_staging.rlc'
+    url = "https://ckan0.cf.opendata.inter.prod-toronto.ca/dataset/9fcff3e1-3737-43cf-b410-05acd615e27b/resource/7e4ac806-4e7a-49d3-81e1-7a14375c9025/download/Red%20Light%20Cameras%20Data.geojson"  
+    return_json = requests.get(url).json()
+    
+    rlcs = return_json['features']
+    rows = []
+    for info in rlcs:
+        # temporary list of properties of one RLC to be appended into the rows list
+        one_rlc = []
+
+        # dive deeper into the json objects
+        properties = info['properties']
+        geom = info['geometry']
+        coords = geom['coordinates']
+
+        # append the values in the same order as in the table
+        one_rlc.append(properties['RLC'])
+        one_rlc.append(properties['TCS'])
+        one_rlc.append(properties['NAME'])
+        one_rlc.append(properties['ADDITIONAL_INFO'])
+        one_rlc.append(properties['MAIN'])
+        one_rlc.append(properties['SIDE1'])
+        one_rlc.append(properties['SIDE2'])
+        one_rlc.append(properties['MID_BLOCK'])
+        one_rlc.append(properties['PRIVATE_ACCESS'])
+
+        # longitude and latitude
+        one_rlc.append(coords[1])
+        one_rlc.append(coords[0])
+
+        one_rlc.append(properties['X'])
+        one_rlc.append(properties['Y'])
+        one_rlc.append(properties['DISTRICT'])
+        one_rlc.append(properties['WARD_1'])
+        one_rlc.append(properties['WARD_2'])
+        one_rlc.append(properties['WARD_3'])
+        one_rlc.append(properties['WARD_4'])
+        one_rlc.append(properties['POLICE_DIVISION_1'])
+        one_rlc.append(properties['POLICE_DIVISION_2'])
+        one_rlc.append(properties['POLICE_DIVISION_3'])
+        one_rlc.append(properties['ACTIVATION_DATE'])
```

Comment:
might be easier having a variable containing all these names and just run `one_rlc.append(properties[ ])` in a for loop.